spec/unit/outputs/elasticsearch/http_client/pool_spec.rb

```diff
@@ -45,46 +45,27 @@
     end
 
     describe "healthcheck url handling" do
-      let(:initial_urls) { [::LogStash::Util::SafeURI.new("http://localhost:9200#{path}")] }
-      let(:healthcheck_path) { "/some/other/path" }
-      let(:absolute_healthcheck_path) { false }
-      let(:path) { "/meh" }
-      let(:options) { super.merge(:absolute_healthcheck_path => absolute_healthcheck_path, :path => path, :healthcheck_path => healthcheck_path) }
-
-      context "when using query params in the initial url" do
-        let(:initial_urls) { [::LogStash::Util::SafeURI.new("http://localhost:9200?q=s")] }
-        it "is removed from the healthcheck request" do
+      let(:initial_urls) { [::LogStash::Util::SafeURI.new("http://localhost:9200")] }
+
+      context "and not setting healthcheck_path" do
+        it "performs the healthcheck to the root" do
           expect(adapter).to receive(:perform_request) do |url, method, req_path, _, _|
-            expect(method).to eq(:head)
```

Comment:
IMO that was an extra concern not related to this test which was checking the path handling of the health check request, the fact that the request is a HEAD is included in another test